src/containers/VisualElement/DisplayVisualElement.jsx

```diff
@@ -49,9 +49,24 @@ const DisplayVisualElement = ({
           language={language}
         />
       );
-    case 'external':
     case 'iframe':
     case 'h5p':
+    case 'external':
+      if (embed.url.includes('youtu'))
```

Comment:
H5p har ikkje url-parameter så vi må ha en ? her.
```suggestion
      if (embed.url?.includes('youtu'))
```